python/src/energuide/element.py

```diff
@@ -0,0 +1,18 @@
+from lxml import etree
+
+
+class Element:
```

Comment:
This change is to help `mypy` catch type errors on functions that take or return `_Element` values - right now it just skips them.